core/src/main/scala/cats/syntax/cartesian.scala

```diff
@@ -17,10 +17,4 @@ abstract class CartesianOps[F[_], A] extends Cartesian.Ops[F, A] {
   final def |@|[B](fb: F[B]): CartesianBuilder[F]#CartesianBuilder2[A, B] =
     new CartesianBuilder[F] |@| self |@| fb
 
-  final def *>[B](fb: F[B])(implicit F: Functor[F]): F[B] =
```

Comment:
Ahh. I see someone put them here originally. This was an error in my view. They should be on Apply.

Reply:
Yeah +1 on that :)